src/electron-main.ts

```diff
@@ -192,11 +191,13 @@ async function setupGlobals(): Promise<void> {
     }
 
     // The tray icon
-    // It's important to call `path.join` so we don't end up with the packaged asar in the final path.
-    const iconFile = `element.${process.platform === 'win32' ? 'ico' : 'png'}`;
-    iconPath = path.join(resPath, "img", iconFile);
+    // It's important to call `path.join` for the bundled assets so we don't end up with the packaged asar in the final path.
+    const icon_path = process.platform === 'win32'
```

Comment:
variables should be `lowerCamelCase`

Reply:
What is the rule for config keys? They seem pretty much evenly split between `snake_case` and `camelCase`; couldn't seem to find a style guide.

Reply:
Config keys are `snake_case`.